src/SDKs/Azure.Base/data-plane/Azure.Base/Azure.Base.props

```diff
@@ -1,11 +1,6 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Project references -->
-  <ItemGroup Condition="'$(UseProjectReferenceToAzureBase)'=='true'">
+  <ItemGroup>
```

Comment:
I guess you did this to get the project easier to build right? :)

Reply:
right